package.json

```diff
@@ -1,45 +1,45 @@
 {
-    "name": "jasmine-node-helpers",
-    "version": "1.0.0",
-    "description": "Public and free software for helping test NodeJS projects using Jasmine-Node.",
-    "main": "/index.js",
-    "scripts": {
-        "test": "istanbul cover jasmine-node --captureExceptions spec/ && codecov && eslint .",
-        "watch": "jasmine-node lib spec/ --autotest --watch"
+  "name": "jasmine-node-helpers",
+  "version": "1.0.0",
```

Comment:
Didn't we publish 1.0.0 already?  If so, this must go up.  Since it's backwards compatible and introducing new features, we should use 1.1.0.